lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -94,6 +97,30 @@ class NoTextInputType extends EngineInputType {
   String get inputmodeAttribute => 'none';
 }
 
+/// Multi-line no text input from system virtual keyboard.
+///
+/// Use this for inputting multiple lines with a customized keyboard.
+///
+/// When Flutter uses a custom virtual keyboard, it sends [TextInputType.none]
+/// with a [forceMultiline] flag to block the system virtual keyboard.
+///
+/// For [MultilineNoTextInputType] (mapped to [TextInputType.none] with 
+/// [forceMultiline] = true), it creates a <textarea> element with the 
+/// inputmode="none" attribute.
+///
+/// For [NoTextInputType] (mapped to [TextInputType.none] with 
+/// [forceMultiline] = false), it creates an <input> element with the 
+/// inputmode="none" attribute.
```

Comment:
@htoor3 
Is the comment clear now? I'm sorry, I'm not good at English. If it's still unclear, could you please write it for me to copy? Thank you.

Reply:
Thank you @Satsrag! It looks good to me, I would just add another line that points to the issue this is fixing like
```
/// See: https://github.com/flutter/flutter/issues/125875
```